binary_encode_override_test.go

```diff
@@ -0,0 +1,44 @@
+package amino_test
+
+import (
+	"github.com/stretchr/testify/assert"
+	amino "github.com/tendermint/go-amino"
+	"testing"
+)
+
+type Thing struct {
+	Name string
+}
+
+// func (thing Thing) MarshalAmino() (string, error) {
```

Comment:
These comments can be removed, they were from earlier testing of `MarshalAmino`/`UnmarshalAmino`